module/migration.js

```diff
@@ -323,3 +320,13 @@ const migrateDarknessPoints = async function () {
     });
   }
 };
+
+export const migrateActorKeyArt = function () {
```

Comment:
Either is we want to create a proper migration or just a utility tool to create a marco like 

`game.yzecoriolis.migrations.migrateActorKeyArt()`